src/services/generateAccessToken.ts

```diff
@@ -0,0 +1,14 @@
+import jwt from 'jsonwebtoken';
+import { Types } from 'mongoose';
+
+const secret: string = process.env.SECRET_KEY || '';
+
+const generateAccessToken = (id: Types.ObjectId, roles: string[]) => {
+    const payload = {
+        id,
+        roles,
+    };
+    return jwt.sign(payload, secret, { expiresIn: '120000ms' });
+};
+
+export default generateAccessToken;
```

Comment:
About all services that you created.
You don't need to create separate files for each method.
Create `authService.ts` and store there all methods related to authentication